### DISCUSSION TITLE
How to select vertices by tag with multiple tags given

### DISCUSSION THREAD

**Discussion by imrefazekas** [vesoft-inc/nebula#5546] [General]
Hello,
I would love to have such expression:
```
MATCH (v:V_ACCOUNT)-[e]-(V_PERSON|V_COMPANY)
RETURN v, e;
```
based on this document: https://neo4j.com/docs/cypher-manual/current/clauses/match/

but Nebula gives syntax error. Cypher support is not complete or I misunderstand how to select vertices belonging to one tag group or another?

**Comment by wey-gu** [comment]
Try with this:

```
MATCH (n:player)-[e]->(n1) WHERE tags(n1)[0] IN ["team", "player"] return n,e LIMIT 3
```
